src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -485,46 +485,43 @@ void KivaManager::readWeatherData(EnergyPlusData &state)
         }
         if (Pos != std::string::npos) LineResult.data.erase(0, Pos + 1);
 
-        {
```

Comment:
There was no need for this to be a SELECT_CASE, it's just checking if a string is another string for one case.  Now, whether this code should actually be in here or not is a question for a different day.